go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
I think one class per file is good, it makes people easy to navigate your code even with limited IDE support. The only case I would prefer to put multiple classes in a single file is probably some message classes. Back to this case, it looks like this class is added for testing purpose, could we add "_test" prefix so that it does not count in go coverage and also it makes sure these test code won't in the production binaries.